Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/ADLSGen2Deploy.java

```diff
@@ -72,7 +72,7 @@ private URI getUploadDir() {
         final String filePath = String.format("%s/%s", dirPath, src.getName());
 
         final ADLSGen2FSOperation op = new ADLSGen2FSOperation(this.http);
-        return op.createDir(dirPath)
+        return op.createDir(dirPath, "0755", "0000")
```

Comment:
Since the access key and oauth share the same code path. I'm wondering `0755` is proper to the folder under user's home.